.github/workflows/test.yml

```diff
@@ -44,6 +44,26 @@ jobs:
         run: build_tools/run_examples.sh
         shell: bash
 
+  test-nodevdeps:
+    needs: code-quality
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
```

Comment:
how would that look like? I don't want to spam the test matrix, and for checking imports either version should do, I thought?

Reply:
> how would that look like

Standard `nox` pattern, e.g. `nox -s name_of_session`

Yes, imports will mostly be same, but just in case different python version gets solved to different versions of core dependencies, their dependencies have a chance to vary.

Probably not very likely,  so can be ignored to avoid complications.

Reply:
ok, so I'll leave it for now.